pyproject.toml

```diff
@@ -31,13 +31,13 @@ classifiers = [
 ]
 dynamic = ["version"]
 dependencies = [
-  "dvc>=3.33.3",
+  "dvc@git+https://github.com/iterative/dvc.git@refs/pull/10303/head",
```

Comment:
You probably didn't wanna push those changes

Reply:
The required DVC PR isn't merged/released yet. I will update this PR once the required version of DVC is released. That can't happen until https://github.com/iterative/studio/pull/9173 is merged and released.